browse-next/src/views/DeviceDiagnosticsSubView.vue

```diff
@@ -454,7 +767,123 @@ const deviceLocationPoints = computed<NamedPoint[]>(() => {
       </div>
     </div>
     <div class="mt-4" v-if="device.type === 'thermal'">
+      <h6 class="mt-4">Device Settings:</h6>
+      <span v-if="settingsLoading">
+        <b-spinner small class="me-2" />
+      </span>
+      <div v-else-if="settings" class="device-settings mt-3">
+        <div><b>Synced:</b> {{ settings.synced ? "Yes" : "No" }}</div>
+        <b-table
+          :items="settingsTable"
+          :fields="fields"
+          striped
+          hover
+          class="settings-table mt-3"
+        >
+          <template #cell(actions)="row">
+            <div v-if="row.item.name === 'Use Low Power Mode'">
+              <b-button
+                @click="handleToggleUseLowPowerMode(row.item)"
+                variant="secondary"
+                >Toggle</b-button
+              >
+            </div>
+            <div v-if="row.item.name === 'Recording Windows'" class="btn-group">
+              <b-button
+                @click="handleSetDefaultRecordingWindows"
+                :variant="
+                  currentWindowsType === 'default' ? 'primary' : 'secondary'
+                "
+                >Default</b-button
+              >
+              <b-button
+                @click="handleSet24HourRecordingWindows"
+                :variant="
+                  currentWindowsType === '24hour' ? 'primary' : 'secondary'
+                "
+                >24 Hours</b-button
+              >
+              <b-button
+                @click="enableCustomRecordingWindows"
```

Comment:
In my testing, `enableCustomRecordingWindows` did nothing, I assume it's because there were not already window settings defined in my device settings, so it returned early?  That case probably wants to be handled.